systemtest/src/main/java/io/strimzi/systemtest/kafkaclients/internalClients/VerifiableClient.java

```diff
@@ -122,8 +122,9 @@ public VerifiableClient(VerifiableClientBuilder verifiableClientBuilder) {
 
         if (clientType == ClientType.CLI_KAFKA_VERIFIABLE_CONSUMER) {
             this.consumerGroupName = verifiableClientBuilder.consumerGroupName;
-            this.clientArgumentMap.put(ClientArgument.GROUP_ID, consumerGroupName);
             this.consumerInstanceId = verifiableClientBuilder.consumerInstanceId;
+            this.clientArgumentMap.put(ClientArgument.GROUP_ID, consumerGroupName);
+            this.clientArgumentMap.put(ClientArgument.GROUP_INSTANCE_ID, consumerInstanceId);
```

Comment:
Why this change? Seems unrelated to spotbugs or checkstyle.

Reply:
Spotbugs reported `consumerGroupName` as unused. Looking at the code I thing it might a bug.
It was partially fixed in https://github.com/strimzi/strimzi-kafka-operator/pull/6156/files#diff-f93d2fac0903e6c67c6a2111f4b65fa4b61a2fd27690459d80d0ae4c6561a70eR126
@fvaleri @see-quick Could you please take a look?

Reply:
Yeah, that actually revealed a bug and I forgot to add it to the arguments map. Thanks.